commissioning.tex

```diff
@@ -1,27 +1,48 @@
 \section{Rubin Observatory Commissioning}
 \label{sec:commissioning}
 
+\subsection{Commissioning Milestones}
+\label{ssec:commissioning-milestones}
+
+Commissioning work is being planned around two major milestones, ``First Photon'' and ``System First Light''. 
```

Comment:
This is more complicated with ComCam. If First photon is for LSTCam, then the period between FP and FL is less than 2 months (see dmtn-232). The text below needs to be reconciled with ComCam. It may be the easiest thing to do is not talk about first photon but just the activities.

Reply:
Added in the ComCam FirstLight milestone